### DISCUSSION TITLE
Running ASB, How to access Bitcoin?

### DISCUSSION THREAD

**Discussion by xhwmerhfity** [comit-network/xmr-btc-swap#1205] [Q&A] [answered]
Running an ASB and can complete test swaps OK. Monero is received by Bob and I can see Bitcoin has moved to Alice and confirmed.

However, how do I access the received Bitcoin?

Running "asb balance" or "asb withdraw-etc" shows the message:

2022-11-09T16:52:41.513090982Z  INFO Initialized tracing level=debug
2022-11-09T16:52:41.51327155Z  INFO Running initial setup for mainnet

? Enter data directory for asb or hit return to use default (/root/.local/share/xmr-btc-swap/asb/mainnet) › 
---
I thought everything was stored in a sqlite database?
Or do I need to create a data directory?

Is there documentation anywhere regarding how this works?

Thanks in advance.

**Comment by binarybaron** [comment, marked as answer]
Perhaps you are running the ASB from another user?

**Reply by xhwmerhfity** [reply]
Thank you for your reply.

I identified the problem was the data directory I entered (when prompted above) was different than the data directory in the config.toml file. Entered the same directory and "balance" and "withdraw-bitcoin" commands work perfectly.

Feel free to leave this QA up if it might be useful to others.

**Comment by binarybaron** [comment]
Perhaps you are running the ASB from another user?
